presto-main/src/main/java/com/facebook/presto/security/FileBasedSystemAccessControl.java

```diff
@@ -104,6 +107,20 @@ public SystemAccessControl create(Map<String, String> config)
         }
     }
 
+    @VisibleForTesting
```

Comment:
Rather than testing this directly, it seems better to use a resource file for the JSON and have the test call `createAccessControl()`, so that the loading goes through the normal code path.